frontend/src/citizen-frontend/calendar/CalendarPage.tsx

```diff
@@ -125,10 +125,15 @@ const CalendarPage = React.memo(function CalendarPage() {
         data.value.children,
         data.value.reservableDays
       )
+      const allReservableDateRanges = Object.keys(
+        data.value.reservableDays
+      ).flatMap((childId) => data.value.reservableDays[childId])
+
       // First reservable day that has no reservations
       const firstReservableEmptyDate = data.value.dailyData.find(
         (day) =>
           earliestReservableDate?.isEqualOrBefore(day.date) &&
+          allReservableDateRanges.find((range) => range.includes(day.date)) &&
```

Comment:
With this change we could just not calculate `earliestReservableDate`. I don't think it has any function since it's calculated from `data.value.reservableDays` anyway.

Reply:
Good point!